src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -348,9 +349,12 @@ def evaluate(
     evaluator_config = _process_evaluator_config(evaluator_config)
     _validate_columns(input_data_df, evaluators, target, evaluator_config)
 
+    # We are checking that the workspace exist as it was done in promptflow-azure.
+    _validate_tracing_uri(trace_destination)

```

Comment:
Here are the things pf azure checks:

    1. the value is a valid ARM resource ID for workspace/project
    2. the resource exists
    3. the resource is an Azure ML workspace or AI project
    4. the workspace Cosmos DB is initialized
  
3 should be straight forward, but not sure how to handle Cosmos DB validation. Is Cosmos DB provisioned by default as part of AI project creation?

Reply:
Cosmos DB is not provisioned by default

Reply:
Per offline discussion removed this code.